bin/git-bug

```diff
@@ -1,14 +1,4 @@
 #!/usr/bin/env bash
 
-if test "$1" = "finish"; then
-  test -z $2 && echo "bug <name> required." 1>&2 && exit 1
-  branch=bug/$2
-  git merge --no-ff $branch && git delete-branch $branch
-else
-  test -z $1 && echo "bug <name> required." 1>&2 && exit 1
-  if test -n "$2"; then
-    echo "too many arguments; if not finishing a bug, only <name> of new bug is expected" && exit 1
-  fi
-  branch=bug/$1
-  git checkout -b $branch
-fi
+ALIAS=bug
+git feature -a $ALIAS $@
```

Comment:
A little suggestion:
Use `"$@"` instead of `$@`, to avoid re-splitting arguments with whitespaces.
Although we don't need it now, it could be helpful for future change.